tuxemon/core/states/combat/combat_menus.py

```diff
@@ -75,6 +84,42 @@ def swap_it(menuitem):
         menu.anchor("right", self.game.screen.get_rect().right)
         menu.monster = self.monster
 
+    def open_item_menu(self):
```

Comment:
move all the logic for displaying menus and items from the item state to the combat state.  the idea is that menus shouldn't have logic to deal with what state they are called from....it should be up to the calling state to determine what the next state does.